src/controllers/index.ts

```diff
@@ -0,0 +1,4 @@
+import { EventController } from './EventController';
+
+export { EventController };
```

Comment:
Why two export statements here? Don't you need only one?

Reply:
there's a reference to EventController within index.ts as im exporting the array of controllers to be used in index.ts and the docs stuff